code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -326,13 +326,16 @@
 		var/obj/structure/spacevine/SV = exposed_obj
 		SV.on_chem_effect(src)
 
-/datum/reagent/toxin/plantbgone/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)
+/datum/reagent/toxin/plantbgone/expose_mob(mob/living/exposed_mob, methods = TOUCH, reac_volume)
```

Comment:
podpeople already handle this in their handle_chemicals

Reply:
This isn't what this proc does.

`expose_mob` is about when something touches you or you ingest it, initially.

`on_mob_life` is what happens every time the mob's body/liver processes the reagents. 

Pod people (and wood golems) have a proces for handling reagents as they are metabolized, but they do not have a process for handling getting *touched* by certain reagents.

Reply:
what about the flipside then

shouldnt all plant biotypes also take some damage when the reagent is being metabolized if its going to damage them on apply?

Reply:
Perhaps, though this becomes a debate over which is more OOP.

Should it be defined on the reagent with a biotype check, or on the species?

With the floral gun, it makes a bit more sense since it's applying to a wide range of mob types.

With reagents---only really humans process reagents (well carbons too, but that's really only xenos).